docs/topic/whentouse.rst

```diff
@@ -0,0 +1,82 @@
+.. _topic/whentouse:
+
+===================================
+When to use The Littlest JupyterHub
+===================================
+
+The Littlest JupyterHub allows you to deploy a JupyterHub serving user sessions
+on a single VM (or your own hardware). The Zero to JupyterHub guide allows you
+to deploy JupyterHub on Kubernetes, making it much more scalable.
+This page is a brief guide to determining which is best for your use-case.
+Many of these ideas were first laid out in a
+`blog post announcing TLJH <http://words.yuvi.in/post/the-littlest-jupyterhub/>`_.
+
+The Littlest JupyterHub
+-----------------------
+
+The Littlest JupyterHub (TLJH) is an opinionated and pre-configured distribution
+to deploy a JupyterHub on a single virtual machine (or your own hardware).
+It is designed to be a more lightweight, flexible, and maintainable solution
+for use-cases where scalability and cost-savings are not a huge concern.
+
+When to use TLJH
+^^^^^^^^^^^^^^^^
+
+* You want to provide easy access to a shared computational resource (CPU, RAM, data, etc)
+* You only need to support up to ~100 people
+* You aren't concerned about `over-provisioning your cluster <https://community.spiceworks.com/cloud/article/overprovisioning-servers-iaas>`_.
+* You don't need production-level security promises for your deployment
+
+TLJH main benefits
+^^^^^^^^^^^^^^^^^^
+
+* Simpler to deploy on many kinds of cloud services
+* Faster to set up and tear down
```

Comment:
this comment has nothing to do with this line. Maybe we should split up the "index.rst" sections into sub-sections rather than having a single "how-to" section